index.html

```diff
@@ -191,9 +192,47 @@ <h2>About</h2>
                 </div>
 
             </section>
+    
+<!-- foooter -->
+<footer class="contact_section">
+    <div class="mobile_logo"></div>
+    
+      <h4 class="title-contact text_header_font">Get Started</h4>
+
+      <p class="section-desc text_body_font">
+        Hello!I am a software developer! I can 
+        help you build a product, feature or website.
+        Take a look of my works. If you like what you see 
+        and have a project you need coded, don't hesitate 
+        and contact me.
+      </p>
+      <form action="https://formspree.io/f/mzbovllg" method="post">
+          <input type="text" name="FullName" placeholder="Enter Your Name" maxlength="30" required>
+          <input type="email" name="Email" placeholder="Enter Your Email" required>
+          <textarea name="Message" placeholder="Enter Your Message"  maxlength="500" required></textarea>
+          <button class="" type="submit" name="submit-btn">Start collaboration</button>
+      </form>
+
+      <div class="bottom_row">
+        <a class="text-button text_body_font resume_color">
+            <p>Get my Resume</p>
+            <div class="arrow_upload"></div>
+        </a>
+
+        <div class="divicon">
+            <ul class="sociallist">
+           <li class="sociallist_link"><a href="#"><img src="img/twitter.png" alt="twitter"></a></li>

```

Comment:
- [ ] Please fix this weird space beside the website so your code looks more responsive consider changing the width of the icons list you set the margin and padding of the body and HTML selector to zero and the width and the height to 100%
![image](https://user-images.githubusercontent.com/74744787/160095545-224fad81-83eb-4bb6-b3f8-04c9b4453727.png)